### DISCUSSION TITLE
⏳ 🔗 New layouts for Section, Container, Block, and Div

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#23622] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi👋

I strongly agree that Elementor should adopt the bricks method in layout. But the problem is that Elementor (the creators of Elementor) want Elementor to always remain simple so that it can be used by the newest users. While professional users prefer to do everything in principle, even if it causes a little difficulty. If you really think so, it will be a serious blow to Elementor.

I hope Elementor developers will make a serious decision about it internally or even poll users. This could shape the future of Elementor.

### Describe the solution you'd like

What I would personally like to see in Elementor's layout is to remove the grid widget and have 4 main layout widgets.

1- **section** > full width - root layout - only one div
2- **container** (Now we have this widget and it controls the width of the entire content) - only one div
3- **Block** (divide the container) - full width - only one div
4- **div**

As I mentioned above, the grid widget will be removed and instead, you will have the display option in all layouts, which includes several options. Flex, Grid, inline, block, inline-block and None.

**As a suggestion, if you think these things are confusing for new users, you can add it as a module in settings so that professional users can enable it and experience and use professional layout. (But the current layout will be as default)**

cheers

**Comment by alanavelino** [comment]
@79ho3ein I completely agree with you, I hope elementor takes us into account and we will be able to see this soon, since that Bricks system is perfect.

**Reply by Henzxon** [reply]
@79ho3ein , I do agree with you as well. This is the very reason why WebFlow can output a clean code, they implement their containers based on how browser expect it to be (not by the users) while keeping the UI easy to understand.

With elementor, there's a lot of run-time optimisation in order to reduce container depth which could be saved for something else instead.

There are three users: (Client, Beginner, Intermediate/Advanced)

The truth is, client prefers to only have control to editing content like text, image, buttons, and title.
Intermediate and Advanced users are now left out of the water (having hard time to explain core concepts properly because the builder build things differently)

Put it simply, it should be:
For advance users, they can assemble from scratch (box model)
For new users, they can import section widget with pre-populated HTML elements like, div block etc so they can drop Heading or HTML right away–keeping the box model layout.

This is the base of all modern web design and a weak foundation would results to weak results.

Just my point of view about this.

**Reply by Henzxon** [reply]
Instead, we are having non-stop development on hosting. So much lost development potential even if you have dedicate team and resources.

**Comment by angelvdc** [comment]
Friend, @79ho3ein without more words to add to your request, I agree with you. 😎

* The only thing I would add to your comment would be to add span rows and columns to the new grid container.

Team @elementor and with that I could return to Elementor again, at this moment I have to use Bricks because Elementor is limited in advanced layout issues. 💔

Come on Elementor team you guys can do this!

**Comment by ivdev17** [comment]
@avivu @nicholaszein There is nothing more to say, this is necessary and I agree.

**Comment by mgperdomo** [comment]
@nicholaszein Definitely a good idea, in addition to improving the experience and not wasting so much time, currently I spend a lot of time adjusting and naming containers to follow that structure, this should be a priority and not fill us with AI and Hosting for that there are many solutions please listen to the community and do not say that you are going to investigate and those things that you always say.

**Reply by nicholaszein** [reply]
@mgperdomo,

AI is a different product, used and requested by many users. Having different products does not mean reducing or not working on others.

Please stop insisting on complaining about Elementor's other products. Focus on discussing the topic in question.

Make sure to read our guidelines.

https://github.com/orgs/elementor/discussions/23196

**Comment by nicholaszein** [comment]
Hello everyone. We are considering adding these new widgets, however due to the current transition, we want to make sure not to confuse users by thinking they are adding something (Sections), but editing something else (Containers).

After we decide Containers are well established, we will definite consider this feature request. Until then, just keep voting and promoting this request.

We are monitoring it and you'll be the first to know if,  and when we decide to add them to our Roadmap.

Kind regards

**Comment by Joliamendi** [comment]
I strongly agree with this suggestion. It would also be great if we could control the amount of padding and margin by default (from site settings) for these 4 layouts.

**Comment by Michaelgimii** [comment]
Elementor Team, let's prioritize and add these widgets with a quick action. You can act quickly. As you added css Grid quickly, when many thought it would be added in 2 years.

**Comment by robertomrtze** [comment]
Team @Elementor this is very necessary I hope to see you soon.

**Comment by saawood13** [comment]
I have been searching and searching for this before I continue on with my website because I came across this issue frequently with my last website. I will have a content container/section/block/whatever we are calling it now that I will use across multiple pages. Sometimes I make minor edits and tweaks to that one container/section. I then have to go and hunt and find every last section and change them manually. I am a little dumb on the technical lingo, but if I understand correctly, this feature request is exactly what I am looking for. 

Is there any word on when this update will come to pass? I will continue on till then but I look forward to making my website easier to manage !!
